LuatixHQ/opencti
# [backend/frontend] Add assignee through automation playbook (#7728)

### Proposed changes

* Add ObjectAssigneeFields in Manipulate Knowledge playbook form
* Add Assignee Input into playbook components for mapping attribute

### Related issues

* https://github.com/OpenCTI-Platform/opencti/issues/7728
* https://github.com/OpenCTI-Platform/client-python/pull/792

### To test
* Add a new playbook 
* Choose to "Listen Knowledge Event" block 
* Choose an event to listen (create/update or both)
* Proceed to filter to a Case (Incident/RFT/RFI/Task)
* Add "Manipulate Knowledge" block
* Select Replace => Assignee => Value of your choice
* Add "Send for Ingestion" block
* Go to a Case and and update/create
* After a moment you should be able to see the new Assignee in this case

FIY: You need to enable your worker to be able to ingest AND to be on the related branch in client-python as it's a multi repository feature
